Topics/Tech_Stacks/Ruby_on_Rails.md

```diff
@@ -0,0 +1,86 @@
+# Introduction to Ruby on Rails
+
+## Objective
+In this guide, we will cover setting up a Rails application and a few basic knowledge about layouts and principles of a Rails application
+
+## Prerequisite
+This guide assumes that you have prior experience with Ruby, a programming language. You might find a steep learning curve if you dive straight into Rails.
+
+Here are some useful resources to learn Ruby:
+- [Official Ruby Website](https://www.ruby-lang.org/en/documentation/)
+- [Ruby Tutorial For Beginners](https://www.rubyguides.com/ruby-tutorial/)
+
+## What is Rails?
+Rails is a framework for web application development in the Ruby programming language. Its relationship to Ruby is similar to the one between Django and Python. The benefit of leveraging Rails is that it allows developers to write less code while still achieving more compared to other languages and frameworks. What makes Rails different from other frameworks is that it is opinionated software, designed with particular rules and preferences on how developers should write code. 
```

Comment:
Maybe explain further of what is Rail and what Rail could accomplish could help.

Reply:
Can you also explain/rephrase "it allows developers to write less code while still achieving more compared to other languages and frameworks."? This sentence looks a bit ambiguous to me.

Reply:
Thank you both for the comments! They are very helpful and I agree! I will address them in my next commit!